build.gradle

```diff
@@ -224,6 +239,18 @@ task clean(type: Delete) {
     }
 }
 
+clean.dependsOn(':src:AirshipBindings.Android.ADM:clean',
```

Comment:
I don't think this is necessary, like with `syncVersion`.

Reply:
Similarly, this was not the case before the change.